src/rpc/handlers/AMMInfo.cpp

```diff
@@ -103,6 +103,9 @@ AMMInfoHandler::process(AMMInfoHandler::Input input, Context const& ctx) const
 
     auto const lgrInfo = std::get<LedgerInfo>(lgrInfoOrStatus);
 
+    auto issue1 = input.issue1;
+    auto issue2 = input.issue2;
```

Comment:
Maybe move them closer to the first usage?

Reply:
Yeah, not sure why i added them up there tbh. 👍